public/_layout_zh.ejs

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html id="top" lang="zh-Hant">
+  <head>
+    <meta charset="utf-8">
+    <title>Crosswalk项目</title>
+    <link rel="shorcut icon" href="/assets/favicon.ico" type="image/x-icon" />
+    <link rel="icon" href="/assets/favicon.ico" type="image/x-icon" />
+    <script>
+      WebFontConfig = {
+        custom: {
+          families: ['Clear Sans'],
+          urls: ['/css/fonts.css']
+        },
+        google: {
+          families: ['Source Code Pro:n4,n6']
+        },
+        timeout: 2000
+      };
+    </script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script async defer src="//ajax.googleapis.com/ajax/libs/webfont/1.5.3/webfont.js"></script>
+    <link rel="stylesheet" href="/css/main.css">
+
+    <meta name="description" content="<%= description %>" />
+    <meta name="author" content="<%= name %>" />
+    <meta name="handheldfriendly" content="true" />
+    <meta name="mobileoptimized" content="320" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="cleartype" content="on" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+
+    <!-- Facebook -->
+    <meta property="og:side_name" content="<%= name %>" />
+    <meta property="og:title" content="<%= title %>" />
+    <meta property="og:url" content="<%= url + '/' + current.path.join('/').replace('index', '') %>" />
+    <meta property="og:description" content="<%= description %>" />
+    <meta property="og:image" content="/assets/crosswalk-og-banner.jpg" />
+
+    <!-- Twitter -->
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:url" content="<%= url + '/' + current.path.join('/').replace('index', '') %>" />
+    <meta name="twitter:title" content="<%= title %>" />
+    <meta name="twitter:description" content="<%= description %>" />
+    <meta name="twitter:site" content="@<%= social.twitter %>" />
```

Comment:
Let's remove the facebook and twitter and replace it with weibo and wechat for Chinese version